src/main/java/uk/gov/hmcts/reform/orgrolemapping/domain/service/RequestMappingService.java

```diff
@@ -32,20 +38,33 @@
 import java.util.Objects;
 import java.util.Set;
 import java.util.UUID;
+import java.util.concurrent.ConcurrentHashMap;
 import java.util.concurrent.atomic.AtomicInteger;
 
 @Service
 @Slf4j
 @AllArgsConstructor
+@NoArgsConstructor
 public class RequestMappingService {
 
+    @Value("${launchdarkly.sdk.environment}")
+    private String environment;
+
+    @Autowired
+    private PersistenceService persistenceService;
 
     public static final String STAFF_ORGANISATIONAL_ROLE_MAPPING = "staff-organisational-role-mapping";
     public static final String AM_ORG_ROLE_MAPPING_SERVICE = "am_org_role_mapping_service";
     public static final String ROLE_ASSIGNMENTS_QUERY_NAME = "getRoleAssignments";
     public static final String ROLE_ASSIGNMENTS_RESULTS_KEY = "roleAssignments";
+
+    @Autowired
```

Comment:
No need of @Autowired as we are injecting the dependencies through constructor with the help of @AllArgsConstructor.

Reply:
This is to satisfy the integration tests and when it load the entire context it try to create object and throws "NullPointerException" We can discuss, if required